main_f1.c

```diff
@@ -128,7 +124,7 @@ board_deinit(void)
 static inline void
 clock_init(void)
 {
-#if defined(INTERFACE_USB)
+#if INTERFACE_USB
 	rcc_clock_setup_in_hsi_out_48mhz();
 #else
 	rcc_clock_setup_in_hsi_out_24mhz();
```

Comment:
I agree it makes sense to always use ```rcc_clock_setup_in_hsi_out_48mhz();```. But a good exercise would be to look at the commit that added the mavstation and see if there are any other implications on choice of frequency.